scripts/build_layers.sh

```diff
@@ -11,6 +11,13 @@ set -e
 LAYER_DIR=".layers"
 LAYER_FILES_PREFIX="datadog_lambda_node"
 NODE_VERSIONS=("10.15" "12.13")
+DATADOG_LAMBDA_VERSION=$(cat package.json | jq -r ".version")
+DD_TRACE_VERSION=$(cat package.json | jq -r ."devDependencies"'."dd-trace"')
+
+sed -i "" -E "s/(datadogLambdaVersion = )\"([0-9]*\.[0-9]*\.[0-9]*)\"/\1\""$DATADOG_LAMBDA_VERSION"\"/" ./src/trace/listener.ts
```

Comment:
So, we could apply this in build_layers but then running just yarn build, (which we do when deploying the npm package), wouldn't work. You could create a new script, say './update_dist_version.sh', and run that as a post-script to yarn build eg. https://gist.github.com/coryhouse/b7ff5e6d1e41b762c35b, and modify the dist directory instead of touching the src code every time.

Reply:
Done!